views_uid2/index.hbs

```diff
@@ -31,12 +31,8 @@
           var buttonItem = $("<span>", {
             "class": "ui-selectmenu-text"
           });
-          if (item.value !== "") {
-            let callingCode = item.label.substring(item.label.indexOf("+"))
-            this._setText(buttonItem, callingCode);
-          } else {
```

Comment:
I believe this block was only ever hit with the placeholder option, since this is where we set the placeholder value. Since we have removed the placeholder option, I'm thinking we should remove the associated code.